base/remote/R/qsub_parallel.R

```diff
@@ -4,6 +4,7 @@
 #' @param files allow submit jobs based on job.sh file paths.
 #' @param prefix used for detecting if jobs are completed or not.
 #' @param sleep time (in second) that we wait each time for the jobs to be completed.
+#' @param hybrid Decide if we want to detect the job completion by both files and job ids on the server or just by the job ids on the server.s
```

Comment:
unclear what hybrid does and how you configure it. Below I see that it's boolean (not mentioned here) but it's not clear what happens when I set this true vs false

Reply:
Updated.